source/includes/sample-config.rst

```diff
@@ -0,0 +1,7 @@
+.. versionadded:: 2.9
+
+A sample :binary:`~bin.mongosqld` :ref:`configuration file
+<config-format>` named ``example-mongosqld-config.yml`` is included
+with the installation package. For instructions on starting |bi-short|
+with a configuration file, see the :binary:`~bin.mongosqld`
+documentation.
```

Comment:
Would suggest a few changes:
```
.. versionadded:: 2.9

To help you get started, a sample :binary:`~bin.mongosqld`
:ref:`configuration file <config-format>` named
``example-mongosqld-config.yml`` is included
with the installation package. To learn how to start |bi-short|
with a configuration file, refer to the :binary:`~bin.mongosqld`
documentation section on the
:doc:`Configuration File </reference/mongosqld#configuration-file>`.
```

Reply:
This should give a call to action and a more direct link to the relevant information.

Reply:
Yes, looks good.